pkgs/development/libraries/lib3mf/default.nix

```diff
@@ -1,26 +1,26 @@
-{ lib, stdenv, fetchFromGitHub, cmake, ninja, libuuid, libossp_uuid, gtest }:
+{ lib, stdenv, fetchFromGitHub, cmake, ninja, libuuid, libossp_uuid }:
 
 stdenv.mkDerivation rec {
   pname = "lib3mf";
-  version = "2.0.0";
+  version = "2.1.1";
 
   src = fetchFromGitHub {
     owner = "3MFConsortium";
     repo = pname;
     rev = "v${version}";
-    sha256 = "0w4d9zvl95g1x3r5nyd6cr27g6fwhhwaivh8a5r1xs5l6if21x19";
+    sha256 = "0qah4dgird4z1l4lhcfijr5fwnclhfxk5br7fwmqcwkws82y8nia";
+    fetchSubmodules = true;
```

Comment:
I think we should unvendor libressl and googletest.

Reply:
Fortunately upstream has added cmake flags to use the system dependencies now, so this was easy.